pom.xml

```diff
@@ -183,6 +183,10 @@
         <dep.objenesis.version>1.3</dep.objenesis.version>
         <dep.slice.version>0.10</dep.slice.version>
         <dep.jmh.version>1.15</dep.jmh.version>
+        <dep.junit.jupiter.version>5.0.0-RC2</dep.junit.jupiter.version>
+        <dep.junit.jupiter.platform.version>1.0.0-RC2</dep.junit.jupiter.platform.version>
```

Comment:
Let's call this `junit-platform` since that matches their Maven artifacts (which don't contain the word "jupiter").